### DISCUSSION TITLE
What is the heading tag in tailwind

### DISCUSSION THREAD

**Discussion by dakshkothari7** [tailwindlabs/tailwindcss#2041] [Help]
PS Let me know what is h1 tag in tailwind css . Because when I am typing `<h1>` In the web browser it shows so small .

**Comment by estevanmaito** [comment]
Tailwind doesn't style any element by default. You style your elements the way you want, using [utilities](https://tailwindcss.com/docs/utility-first).

If you want to add styles for all the `h1`, for example, you can use [this approach](https://tailwindcss.com/docs/adding-base-styles#using-css)
